lib/landing/landing.dart

```diff
@@ -1,19 +1,19 @@
 import 'package:flutter/material.dart';
 
 import '../constants/colors.dart';
-import 'main_presenter.dart';
-import 'main_view.dart';
+import 'landing_presenter.dart';
```

Comment:
what is landing?

Reply:
Landing is the Widget that controls switching between Home, Events, and Profile. It's called Landing because it's the place that users land after signing up/in. Visually, it's the bottom navigation bar with a transparent top because it shows one of Home/Events/Profile.